resources/views/inc/dashboard-head.blade.php

```diff
@@ -8,24 +8,24 @@
                 @endphp
 
                 @if(($month == 12 && $day == 31) || ($month == 1 && $day == 1))
-                    <img width="100" src="/photos/logos/ztl_logo_fireworks-black.png">
+                    <img src="/photos/logos/ztl_logo_fireworks-black.png">
```

Comment:
Set by the css selector `.navbar-brand img` on line 210 of app.scss

Reply:
This is bad practice, and can lead to confusing issues - in most modern browsers, the css `width` selector is ignored specifically on images, so we should set these here too. I know it's annoying, but earlier today I spent like 4 hours fighting with images deciding on their own sizes because of this

Reply:
c/w